h/activity/query.py

```diff
@@ -165,8 +165,8 @@ def load_documents(query):
 @newrelic.agent.function_trace()
 def _execute_search(request, query, page_size):
     search = Search(request, stats=request.stats)
-    search.append_filter(AuthorityFilter(authority=request.authority))
-    search.append_filter(TopLevelAnnotationsFilter())
+    search.append_qualifier(AuthorityFilter(authority=request.authority))
+    search.append_qualifier(TopLevelAnnotationsFilter())
```

Comment:
Why not use the separate_replies=True feature instead of appending a qualifier?

Reply:
^ I realized this later last night after I thought about it for a while. I wonder where the seperate_replies feature is used today...I'm not a huge fan of appending extra filters inside the run query method as I think it is deceitful and non explicit. I wonder if it would be better to just have a do_not_search_for_replies boolean instead?

Reply:
replies_limit is not used currently-I'm tempted to remove it, `_seperate_replies` is exposed in our api and potentially used there-the question is, is anyone actually using it when they call our api cause otherwise, it too is not used.

Reply:
there is no mention of what appears to be private _seperate_replies in our api either so very tempted to remove this as well.